app/grandchallenge/products/templatetags/products_tags.py

```diff
@@ -22,23 +22,33 @@ def navbar(context):
                 "active": url in ["company-list", "company-detail"],
                 "title": "Companies",
             },
-            {
-                "url": "project-air",
-                "active": url == "project-air",
-                "title": "Project AIR",
-            },
             {
                 "url": "blogs-list",
-                "active": url == "blogs-list",
+                "active": url in ["blogs-list", "blogs-detail"],
                 "title": "Blogs",
             },
+        ],
+        "contact": {
```

Comment:
Why not add these tho items as well, as they're also items. For About you could then add a `subitems` key and simply add the html for the subitems in an `{% if %}` block in the template.

Reply:
Let's have a look at it together tomorrow!

Reply:
fixed